test/mortgage_e2e.ipynb

```diff
@@ -0,0 +1,1123 @@
+    "    \"\"\" Loads acquisition data\n",
+    "\n",
+    "    Returns\n",
+    "    -------\n",
+    "    Dask GPU DataFrame\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    cols = [\n",
+    "        'loan_id', 'orig_channel', 'seller_name', 'orig_interest_rate', 'orig_upb', 'orig_loan_term', \n",
+    "        'orig_date', 'first_pay_date', 'orig_ltv', 'orig_cltv', 'num_borrowers', 'dti', 'borrower_credit_score', \n",
+    "        'first_home_buyer', 'loan_purpose', 'property_type', 'num_units', 'occupancy_status', 'property_state',\n",
+    "        'zip', 'mortgage_insurance_percent', 'product_type', 'coborrow_credit_score', 'mortgage_insurance_type', \n",
+    "        'relocation_mortgage_indicator'\n",
+    "    ]\n",
+    "    \n",
+    "    dtypes = OrderedDict([\n",
+    "        (\"loan_id\", \"int64\"),\n",
+    "        (\"orig_channel\", \"str\"),\n",
+    "        (\"seller_name\", \"str\"),\n",
+    "        (\"orig_interest_rate\", \"float64\"),\n",
+    "        (\"orig_upb\", \"int64\"),\n",
+    "        (\"orig_loan_term\", \"int64\"),\n",
+    "        (\"orig_date\", \"date\"),\n",
+    "        (\"first_pay_date\", \"date\"),\n",
+    "        (\"orig_ltv\", \"float64\"),\n",
+    "        (\"orig_cltv\", \"float64\"),\n",
+    "        (\"num_borrowers\", \"float64\"),\n",
+    "        (\"dti\", \"float64\"),\n",
+    "        (\"borrower_credit_score\", \"float64\"),\n",
+    "        (\"first_home_buyer\", \"str\"),\n",
+    "        (\"loan_purpose\", \"str\"),\n",
+    "        (\"property_type\", \"str\"),\n",
+    "        (\"num_units\", \"int64\"),\n",
+    "        (\"occupancy_status\", \"str\"),\n",
+    "        (\"property_state\", \"str\"),\n",
+    "        (\"zip\", \"int64\"),\n",
+    "        (\"mortgage_insurance_percent\", \"float64\"),\n",
+    "        (\"product_type\", \"category\"),\n",
+    "        (\"coborrow_credit_score\", \"float64\"),\n",
+    "        (\"mortgage_insurance_type\", \"float64\"),\n",
+    "        (\"relocation_mortgage_indicator\", \"str\")\n",
+    "    ])\n",
+    "    \n",
+    "    print(acquisition_path)\n",
+    "    adf = dask_cudf.read_csv(acquisition_path, names=cols, delimiter='|', dtype=list(dtypes.values()), header= True, npartitions = n_partitions)\n",
+    "    return adf\n",
+    "\n",
+    "def gpu_load_names(**kwargs):\n",
+    "    \"\"\" Loads names used for renaming the banks\n",
+    "    \n",
+    "    Returns\n",
+    "    -------\n",
+    "    Dask GPU DataFrame\n",
+    "    \"\"\"\n",
+    "\n",
+    "    cols = [\n",
+    "        'seller_name', 'new'\n",
+    "    ]\n",
+    "    \n",
+    "    dtypes = OrderedDict([\n",
+    "        (\"seller_name\", \"str\"),\n",
+    "        (\"new\", \"category\"),\n",
+    "    ])\n",
+    "    ndf = dask_cudf.read_csv(col_names_path, names=cols, delimiter='|', dtype=list(dtypes.values()), header= True, npartitions = n_partitions)\n",
```

Comment:
Acquisition and names might be small enough where we get away with 1 partition instead of `n`.

Reply:
they are.  will change and test